go.mod

```diff
@@ -2,7 +2,9 @@ module github.com/project-flogo/flow
 
 require (
 	github.com/julienschmidt/httprouter v1.3.0
+	github.com/magiconair/properties v1.8.1
 	github.com/project-flogo/core v0.10.1
```

Comment:
I don't see any new dependence requires for these changes.  Have I missed something here?